schema/team.schema.json

```diff
@@ -35,9 +35,6 @@
       },
       "description": {
         "type": "string"
-      },
-      "hide_on_team_page": {
```

Comment:
It wasn't clear to me that we were hiding interim folks on the org chart or on team pages? But please correct me if I'm wrong. I was looking at [this page](https://handbook.sourcegraph.com/departments/product-engineering/product/team) for reference.